.github/workflows/check-outdated-trusted-root.yml

```diff
@@ -0,0 +1,24 @@
+name: "Check for outdated trusted root certificates"
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
```

Comment:
Missing permissions section: I believe all repositories in the php/* organization are configured with read-only GHA permissions by default. Thus this is unable to create the issue. You'll need:

```
permissions:
  issues: write
```

see: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token

Reply:
Thanks! Good catch, tested it on my own fork, but that doesn't have that restriction I think :) #237

Reply:
Quite possible. Organization-owned repositories behave a little differently. You can sync up the permissions here:

![image](https://github.com/user-attachments/assets/06258640-47a3-4cfd-8ef6-214247584045)

That would likely be at https://github.com/asgrim/pie/settings/actions